docs/getting-started/contributors.md

```diff
@@ -1,6 +1,6 @@
 ---
-id: development-environment
-title: Development Environment
+id: contributors
+title: Contributors
```

Comment:
Changing this ID will require a few more tweaks to prevent local broken links I think...

![image](https://user-images.githubusercontent.com/33203301/108017961-538e6680-6fe4-11eb-88e2-99f9fa97afad.png)